src/main/java/db/Storage.java

```diff
@@ -0,0 +1,11 @@
+package db;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import model.FruitTransaction;
+
+public class Storage {
+    public static final List<FruitTransaction> FRUIT_TRANSACTIONS = new ArrayList<>();
+    public static final HashMap<String, Integer> CURRENT_BALANCE_BY_FRUIT = new HashMap<>();
```

Comment:
You should not save a list of FruitTransaction
and use interface Map instead of current implementation
```suggestion
    public static Map<String, Integer> fruits = new HashMap<>();
```

Reply:
fixed